### ISSUE TITLE
Add UI Single Page Application for the light-config-server

### ISSUE THREAD

**Issue by stevehu** [networknt/light-config-server#6]
We need a single page application running on browsers to manage the data on the light-config-server for each individual service. 

The security will be integrated with the light-portal that is leveraging the light-oauth2 services.

**Comment by miklish** [comment]
That's an option. The other option is to manage the data via the UI provided by the storage provider (e.g.: Vault). light-config-server would then just be used to pull that data from the storage provider and make it available to services when they startup.

**Comment by stevehu** [comment]
Agreed. However, we have to support more backend storage providers and some of them won't have the same level of management UI as Vault. A generic UI for the config server is needed anyway in my opinion.
